README.md

```diff
@@ -316,3 +316,44 @@ except requests.exceptions.HTTPError as exc:
     print(exc.response.json())
 ```
 
+### Deposit
+
+#### Ethereum Deposit
+
+There are two ways to make a deposit on the Ethereum network:
+
+<!-- 1. Using ETH Private Key and RPC URL: This approach utilizes your ETH private key and an rpcUrl (e.g., from Infura or Alchemy).
+2. Custom Provider and Signer: This method involves creating your provider and signer using web3.py. You'll also need the stark_public_key. -->
+
+#### Using Custom Provider and Signer:
+
+This method involves using a custom provider and signer, which can be created using the web3.py library. The `stark_public_key` mentioned in the code should be obtained using the steps described in the [Create L2 Key Pair](#create-l2-key-pair) section of the nodejs connector(https://github.com/tanx-libs/tanx-connector-nodejs#create-l2-key-pair). Here's the code snippet for this method:
```

Comment:
Create L2 Key Pair points to the same repo. It should point to the nodejs connector